occult/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   archive: ^4.0.2
   path: ^1.9.1
   xml: ^6.5.0
+  universal_io: ^2.2.2
```

Comment:
You dont need universal IO. Please just use dart:io which is included for you. Occult CLI will never run on a web browser remove this dep and use dart:io